check_pgactivity

```diff
@@ -5551,6 +5558,18 @@ sub check_streaming_delta {
 
     @rs = @{ query_ver( $hosts[0], %queries ) };
 
+    my $index = 0;
+    if (@exclude){
+        foreach my $line (@rs){
+	    foreach my $exclude_re (@exclude){
```

Comment:
The total of slaves is calculated on line 5573

`push @perfdata => [ '# of slaves', scalar @rs || 0 ];
`

If I exclude my pattern on line 5587 The number '#of slaves' will differ from the number of returned lines.
My goal is to exclude the pattern from every results.